src/main/java/com/google/cloud/flink/bigquery/ArrowDeserializationSchema.java

```diff
@@ -88,7 +86,9 @@ public T deserialize(byte[] responseByteMessage) throws IOException {
     return (T) root;
   }
 
-  private void initializeArrow() {
+  private void initializeArrow() throws IOException {
+    Schema schema = getSchema(schemaJsonString);
+
     Preconditions.checkNotNull(schema);
```

Comment:
getSchema cannot return null, so the check is redundant

Reply:
removed the redundant check